tongueroo/jets
# add apex domain support

This is a 🙋‍♂️ feature or enhancement.



## Summary

Add apex domain support. Docs: https://rubyonjets.com/docs/routing/custom-domain/  Example:

```ruby
Jets.application.configure do
  config.domain.cert_arn = "arn:aws:acm:us-west-2:112233445577:certificate/8d8919ce-a710-4050-976b-b33da991e7e8" # String
  config.domain.hosted_zone_name = "coolapp.com" # String
  config.domain.name = "coolapp.com"
  config.domain.apex = true # # apex or naked domain. Use AliasTarget with an A record instead of a CNAME
end
```
## How to Test

Set `config.domain.apex = true` and deploy. Verify that the apex domain name has been created as an AliasTarget.

## Version Changes

Patch
